neo.UnitTests/Ledger/UT_TrimmedBlock.cs

```diff
@@ -0,0 +1,129 @@
+﻿using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Neo.IO;
+using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
+using Neo.VM;
+using System;
+using System.IO;
+
+namespace Neo.UnitTests.Ledger
+{
+    [TestClass]
+    public class UT_TrimmedBlock
+    {
+        public static TrimmedBlock GetTrimmedBlockWithNoTransaction()
+        {
+            return new TrimmedBlock
+            {
+                ConsensusData = new ConsensusData(),
+                MerkleRoot = UInt256.Parse("0xa400ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff02"),
+                PrevHash = UInt256.Parse("0xa400ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff01"),
+                Timestamp = new DateTime(1988, 06, 01, 0, 0, 0, DateTimeKind.Utc).ToTimestamp(),
+                Index = 1,
+                NextConsensus = UInt160.Parse("0xa400ff00ff00ff00ff00ff00ff00ff00ff00ff01"),
+                Witness = new Witness
+                {
+                    InvocationScript = new byte[0],
+                    VerificationScript = new[] { (byte)OpCode.PUSHT }
+                },
+                Hashes = new UInt256[0]
+            };
+        }
+
+        [TestMethod]
+        public void TestGetIsBlock()
+        {
+            TrimmedBlock block = GetTrimmedBlockWithNoTransaction();
+            block.Hashes = new UInt256[] { TestUtils.GetTransaction().Hash };
+            block.IsBlock.Should().BeTrue();
+        }
+
+        [TestMethod]
+        public void TestGetBlock()
+        {
+            var cache = new TestDataCache<UInt256, TransactionState>();
+            var tx1 = TestUtils.CreateRandomHashTransaction();
```

Comment:
@eryeer, if we change the order or add a new call of `TestRandom` before this function, would this test become invalid because of different hash generation?

Reply:
The test is still valid if change the order because we don't check the transaction hash in the end Haha... But I think it is better not use the random hash to create transactions here. Thanks for your reminding.